nifi-assembly/src/main/assembly/extension-manifests.xml

```diff
@@ -0,0 +1,28 @@
+<?xml version="1.0"?>
+<!--
+  Licensed to the Apache Software Foundation (ASF) under one or more
+  contributor license agreements.  See the NOTICE file distributed with
+  this work for additional information regarding copyright ownership.
+  The ASF licenses this file to You under the Apache License, Version 2.0
+  (the "License"); you may not use this file except in compliance with
+  the License.  You may obtain a copy of the License at
+      http://www.apache.org/licenses/LICENSE-2.0
+  Unless required by applicable law or agreed to in writing, software
+  distributed under the License is distributed on an "AS IS" BASIS,
+  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  See the License for the specific language governing permissions and
+  limitations under the License.
+-->
+<assembly>
+    <id>manifests</id>
+    <includeBaseDirectory>true</includeBaseDirectory>
+    <baseDirectory>nifi-${project.version}-manifests</baseDirectory>
+
```

Comment:
Should define the format here. Also, I think including the `dir` format is redundant. e can probably do without it.
```suggestion

    <formats>
        <format>zip</format>
    </formats>

```